src/utils.ts

```diff
@@ -14,6 +14,14 @@ export type SimpleTarget = {
   preserveTimestamps: boolean
 }
 
+function renameTarget(target, rename, src) {
+  const parsedPath = path.parse(target)
```

Comment:
Is this correct to pass `target`? I wonder if this should be `src`.

Reply:
I copy/paste it from `rollup-plugin-copy`  (https://github.com/vladshcherbin/rollup-plugin-copy/blob/507bf5e99aa2c6d0d858821e627cb7617a1d9a6d/src/index.js#L19-L25). I guess it's correct :-)